### PR TITLE
Add Wombat Exchange DEX & Vela Exchange Perp

### PR DESCRIPTION
**Pull request by chain-l**
Arbitrum One

### CODE REVIEW

File: models/_sector/dex/trades/arbitrum/platforms/wombat_exchange_arbitrum_base_trades.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{{
+    config(
+        schema = 'wombatexchange_arbitrum',
```
Comment (Hosuke) [inline models/_sector/dex/trades/arbitrum/platforms/wombat_exchange_arbitrum_base_trades.sql:3]:
```suggestion
        schema = 'wombat_exchange_arbitrum',
```

File: seeds/_sector/dex/_schema.yml [hunk 2]
```diff
@@ -3002,6 +3002,36 @@ seeds:
...
+        version: varchar
+        tx_hash: varbinary
+        evt_index: uint256
+        block_number: uint256
+        token_bought_address: varbinary
+        token_sold_address: varbinary
+        token_bought_amount_raw: uint256
+        token_sold_amount_raw: uint256
+        block_date: timestamp
+
```
Comment (Hosuke) [inline seeds/_sector/dex/_schema.yml:3019]:
```suggestion
```

File: models/vela_exchange/arbitrum/vela_exchange_arbitrum_perpetual_trades.sql
Comment (Hosuke) [inline models/vela_exchange/arbitrum/vela_exchange_arbitrum_perpetual_trades.sql:1]: Do you want to add this spell into [models/perpetual/perpetual_trades.sql](https://github.com/duneanalytics/spellbook/blob/main/models/perpetual/perpetual_trades.sql)?

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @chain-l 

[Sample wombat_exchange_arbitrum_base_trades]()
[Sample vela_exchange_arbitrum_perpetual_trades]()

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**